code/app.js

```diff
@@ -0,0 +1,19 @@
+
+
+// Faq Accordion // 
+
+const acc = document.getElementsByClassName("accordion");
+let i;
+
+for (i = 0; i < acc.length; i++) {
+  acc[i].addEventListener("click", function() {
+    this.classList.toggle("active");
+    var panel = this.nextElementSibling;
```

Comment:
We should avoid using `var`, (at least in this context) - `const` is what you should aim for over here